### PR TITLE
SL Micro 6.0: Deploy using install_salt_bundle = false, as we do for SLE Micro

### PR DESCRIPTION
**Pull request by srbarrios**
Otherwise, the bootstrap will fail.

### CODE REVIEW

File: terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf [hunk 1]
```diff
@@ -870,6 +870,7 @@ module "slmicro60-minion" {
 // WORKAROUND: Does not work in sumaform, yet
 //  additional_packages = [ "venv-salt-minion" ]
 //  install_salt_bundle = true
+  install_salt_bundle = false
```
Comment (nodeg) [inline terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf:873]:
```suggestion
  install_salt_bundle = false
```

File: terracumber_config/tf_files/SUSEManager-5.0-build-validation-PRV.tf [hunk 2]
```diff
@@ -1091,6 +1091,7 @@ module "slmicro60-minion" {
 // WORKAROUND: Does not work in sumaform, yet
 //  additional_packages = [ "venv-salt-minion" ]
 //  install_salt_bundle = true
+  install_salt_bundle = false
```
Comment (nodeg) [inline terracumber_config/tf_files/SUSEManager-5.0-build-validation-PRV.tf:1094]:
```suggestion
  install_salt_bundle = false
```

**Review by nodeg — APPROVED** [review]
Some small suggestions. Otherwise LGTM.

File: terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf (hunk 1 above)
Comment (srbarrios) [reply terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf:873]: For now, I prefer to keep the whole block commented:
```
// WORKAROUND: Does not work in sumaform, yet
//  additional_packages = [ "venv-salt-minion" ]
//  install_salt_bundle = true
```

Comment (nodeg) [reply terracumber_config/tf_files/SUSEManager-5.0-build-validation-NUE.tf:873]: OK, fine with me.

**Pull request merged**